doc/source/EDBAPI/SiWave.rst

```diff
@@ -0,0 +1,31 @@
+Siwave manager
+==============
+`Siwave <https://www.ansys.com/it-it/products/electronics/ansys-siwave>`_ is a specialized tool for power integrity,
+signal integrity and EMI analysis of IC packages and PCB.
+Solves power delivery systems and high-speed channels in electronic devices.
+It can be accessed from pyaedt in Windows only.
+All Setups can be implemented through EDB API.
```

Comment:
```suggestion
`Siwave <https://www.ansys.com/it-it/products/electronics/ansys-siwave>`_ is a specialized tool for power integrity,
signal integrity, and EMI analysis of IC packages and PCBs. This tool solves power delivery systems and high-speed channels in electronic devices. It can be accessed from PyAEDT in Windows only. All setups can be implemented through EDB API.
```